### PR TITLE
fixed empty state appears when loading the items

### PR DESCRIPTION
**Pull request by Ahmad-AbuOsbeh**
Jira ticket: https://buildfire.atlassian.net/browse/PT-5158

### CODE REVIEW

File: widget/templates/layouts/list-4.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
             <p ng-if="WidgetHome.showDescription()"
                 ng-bind-html="WidgetHome.media.data.content.descriptionHTML | safeHtml"></p>
         </div>
-        <div id="emptyContainer" class="padded"
+        <div id="emptyContainer" style="display: none;" class="padded"
```
[the remark this answers is not in the document]
Comment (Ahmad-AbuOsbeh) [reply widget/templates/layouts/list-4.html:11]: cuz I want them by default to be displayed none; after that I want them to appear. and they already have some styling by "emptyContainer" id, and want display none to be more important first

[the remark this answers is not in the document]
Comment (Ahmad-AbuOsbeh) [reply widget/templates/layouts/list-4.html:11]: i know i can do that by class and give !important thing but i think both ways will work

[the remark this answers is not in the document]
Comment (mh7777777) [reply widget/templates/layouts/list-4.html:11]: you can set important to !important to override display property.

[the remark this answers is not in the document]
Comment (Ahmad-AbuOsbeh) [reply widget/templates/layouts/list-4.html]: done

**Review by mh7777777 — APPROVED** [review]

**Pull request merged**